src/EdFi.AnalyticsMiddleTier.Tests/Dimensions/When_querying_the_DemographicDim_view.cs

```diff
@@ -383,6 +410,15 @@ public void Then_should_have_DemographicLabel()
                         $"{TestCasesFolder}.{_caseIdentifier}_DemographicLabel_should_be_Dominant_language.xml");
                 testResult.success.ShouldBe(true, testResult.errorMessage);
             }
+
+            [Test]
+            public void Then_ShortDescription_should_be_Dominant_language()
+            {
+                (bool success, string errorMessage) testResult =
+                    DataStandard.RunTestCase<DemographicDim>(
+                        $"{TestCasesFolder}.{_caseIdentifier}_DemographicLabel_ShortDescription_should_be_Dominant_language.xml");
```

Comment:
Remove the _DemographicLabel text. It should be _ShortDescription_should_be_Dominant_language. The same for all the ocurrencies.